packages/xo-web/src/xo-app/logs/log-alert-body.js

```diff
@@ -217,131 +238,150 @@ export default [
         />
         <br />
         <ul className='list-group'>
-          {map(state.filteredTaskLogs, vmTaskLog => (
-            <li key={vmTaskLog.data.id} className='list-group-item'>
-              {renderXoItemFromId(vmTaskLog.data.id)} ({vmTaskLog.data.id.slice(
-                4,
-                8
-              )}){' '}
-              <TaskStateInfos
-                status={getTaskStatus(vmTaskLog, state.isJobRunning)}
-              />
-              <ul>
-                {map(logs[vmTaskLog.taskId], subTaskLog => (
-                  <li key={subTaskLog.taskId}>
-                    {subTaskLog.message === 'snapshot' ? (
-                      <span>
-                        <Icon icon='task' /> {_('snapshotVmLabel')}
-                      </span>
-                    ) : subTaskLog.data.type === 'remote' ? (
-                      <span>
-                        {get(remotes, subTaskLog.data.id) !== undefined
-                          ? renderXoItem({
-                              type: 'remote',
-                              value: remotes[subTaskLog.data.id],
-                            })
-                          : _('errorNoSuchItem')}{' '}
-                        ({subTaskLog.data.id.slice(4, 8)})
-                      </span>
-                    ) : (
-                      <span>
-                        {renderXoItemFromId(subTaskLog.data.id)} ({subTaskLog.data.id.slice(
-                          4,
-                          8
-                        )})
-                      </span>
-                    )}{' '}
-                    <TaskStateInfos
-                      status={getSubTaskStatus(subTaskLog, state.isJobRunning)}
-                    />
-                    <br />
-                    {subTaskLog.status === 'failure' && (
-                      <Copiable
-                        tagName='p'
-                        data={JSON.stringify(subTaskLog.result, null, 2)}
-                      >
-                        {_.keyValue(
-                          _('taskError'),
-                          <span className={'text-danger'}>
-                            {subTaskLog.result.message}
-                          </span>
+          {map(state.filteredTaskLogs, vmTaskLog => {
+            const vmStatus = getVmStatus(
+              vmTaskLog,
+              logs[vmTaskLog.taskId],
+              state.isJobRunning
+            )
+            return (
+              <li key={vmTaskLog.data.id} className='list-group-item'>
+                {renderXoItemFromId(vmTaskLog.data.id)} ({vmTaskLog.data.id.slice(
+                  4,
+                  8
+                )}) <TaskStateInfos status={vmStatus} />{' '}
+                {vmStatus === 'failure' && (
+                  <ActionButton
+                    handler={effects.restartVmJob}
```

Comment:
The job should be ran with the same schedule, maybe we need to add it to the log of this run.